src/main/java/org/folio/consortia/service/FolioTenantService.java

```diff
@@ -59,4 +78,13 @@ protected boolean tenantExists() {
       )
     );
   }
+
+  private CustomField createCustomFieldObject() {
```

Comment:
you can move this field to constants like this and remove other constants:

  private static final CustomField ORIGINAL_TENANT_ID_CUSTOM_FIELD =
    CustomField.builder()
      .name("originaltenantid")
      .entityType("user")
      .helpText("id of tenant where user created originally")
      .type(Type.TEXTBOX_LONG)
      .visible(false)
      .build();

Reply:
fixed